brokerapi/brokers/stackdriver_trace/broker.go

```diff
@@ -56,7 +48,7 @@ func (b *StackdriverTraceBroker) Bind(instanceID, bindingID string, details mode
 	details.Parameters["role"] = "cloudtrace.agent"
 
 	// Create account
-	newBinding, err := b.ServiceAccountManager.CreateAccountInGoogle(instanceID, bindingID, details, models.ServiceInstanceDetails{})
+	newBinding, err := b.AccountManager.CreateAccountInGoogle(instanceID, bindingID, details, models.ServiceInstanceDetails{})
```

Comment:
I guess my point is that these brokers are meant to be pretty generic. It shouldn't really matter what kind of account. And the kind of account *is* indicated on creation of the broker. And to have a consistent account manager interface, a generic name is needed anyway.